addons/web/static/src/js/views/kanban/kanban_renderer_mobile.js

```diff
@@ -142,38 +143,61 @@ KanbanRenderer.include({
                    '.o_kanban_group.o_current')
                 .removeClass('o_current');
             this.$('.o_kanban_group[data-id="' + columnID + '"], ' +
-                  '.o_kanban_mobile_tab[data-id="' + columnID + '"]')
+                   '.o_kanban_mobile_tab[data-id="' + columnID + '"]')
                 .addClass('o_current');
         }
     },
     /**
-     * Update the tabs positions
+     * Compute the justify content of the kanban tab headers
      * @private
-     * @param {boolean} [animate=false] set to true to animate
      */
-    _computeTabPosition: function (animate) {
+    _computeTabHeaderJustifyContent: function () {
         if (this.widgets && this.widgets.length) {
             var self = this;
+            // Use to compute the sum of the outerWidth of all tab
+            var widthChilds = this.widgets.map(function (column) {
+                return self.$('.o_kanban_mobile_tab[data-id="' + (column.id || column.db_id) + '"]').outerWidth();
+            }).reduce(function (a, b) {
+                return a + b;
+            });
```

Comment:
missing initial value

Reply:
Also, why not combining the map & reduce (as the map's code is quite simple)?

Reply:
Just for the code readability:
 
1. Compute the outer width of the tabs
2. Made the sum of all outer width

But if you prefer, I can merge these two step